WalletWasabi.Fluent/ViewModels/CoinControl/Core/TreeNodeSorting.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl.Core;
+
+public static class TreeNodeSorting
```

Comment:
I think sorting was usually include in same view model class (see history item etc.) and not in separate class.

Reply:
1. I favor smaller classes with well-defined responsibilities. Easier to read and maintain.
2. It's better to have separate classes that have different reasons to change. In this case, it's pretty evident that the sorting code is a kind of utility code that doesn't have any relationship with the logic of the VM.